File: ExcelToWord/ExcelToWord/Excel to PDF .Net (5.4.1.20)/Code samples/CSharp/How to activate the product/Sample.cs
using System;
using System.IO;
using SautinSoft;

namespace Sample
{
    class Sample
    {
        static void Main(string[] args)
        {
            // Steps to activate the component:

            // 1. Open your project in Visual Studio.
            // 2. Remove the reference to the "SautinSoft.ExcelToPdf.dll".
            // 3. Replace the old file "SautinSoft.ExcelToPdf.dll" by the new "SautinSoft.ExcelToPdf.dll".
            //    (You can find the new "SautinSoft.ExcelToPdf.dll" inside "Bin" in "exceltopdf_net.zip").
            // 4. Add new reference to the new file "SautinSoft.ExcelToPdf.dll".
            // 5. Fill the property 'Serial' by your serial (Activation key):

            string myKey = "1234567890";
            
            // Convert Excel file to PDF file
            ExcelToPdf x = new ExcelToPdf();

            x.Serial = myKey;

            // Set PDF as output format.
            x.OutputFormat = SautinSoft.ExcelToPdf.eOutputFormat.Pdf;

            string excelFile = Path.GetFullPath(@"..\..\test.xls");
            string pdfFile = Path.ChangeExtension(excelFile, ".pdf"); ;

            try
            {
                x.ConvertFile(excelFile, pdfFile);
                System.Diagnostics.Process.Start(new System.Diagnostics.ProcessStartInfo(pdfFile) { UseShellExecute = true });
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
                Console.ReadLine();
            }
        }
    }
}

File: ExcelToWord/ExcelToWord/Excel to PDF .Net (5.4.1.20)/Code samples/VB.Net/How to activate the product/Sample.vb
Imports System
Imports System.IO
Imports SautinSoft

Module Sample

    Sub Main()
        ' Steps to activate the component:

        ' 1. Open your project in Visual Studio.
        ' 2. Remove the reference to the "SautinSoft.ExcelToPdf.dll".
        ' 3. Replace the old file "SautinSoft.ExcelToPdf.dll" by the new "SautinSoft.ExcelToPdf.dll".
        '    (You can find the new "SautinSoft.ExcelToPdf.dll" inside "Bin" in "exceltopdf_net.zip").
        ' 4. Add new reference to the new file "SautinSoft.ExcelToPdf.dll".
        ' 5. Fill the property 'Serial' by your serial (Activation key):

        Dim myKey As String = "1234567890"

        ' Convert Excel file to PDF file
        Dim x As New ExcelToPdf()

        x.Serial = myKey

        ' Set PDF as output format.
        x.OutputFormat = SautinSoft.ExcelToPdf.eOutputFormat.Pdf

        Dim excelFile As String = Path.GetFullPath("..\test.xls")
        Dim pdfFile As String = Path.ChangeExtension(excelFile, ".pdf")

        Try
            x.ConvertFile(excelFile, pdfFile)
            System.Diagnostics.Process.Start(New System.Diagnostics.ProcessStartInfo(pdfFile) With {.UseShellExecute = True})
        Catch ex As Exception
            Console.WriteLine(ex.Message)
            Console.ReadLine()
        End Try
    End Sub
End Module
